cfgov/unprocessed/css/organisms/header.less

```diff
@@ -32,16 +32,16 @@
                 <div class="m-global-search"></div>
                 <a class="o-header_logo" href="/">
                     <img class="o-header_logo-img"
-                         src="{{ static('img/bcfp-logo_240x50.png') }}"
-                         srcset="{{ static('img/bcfp-logo_160x34.png') }} 160w,
-                                 {{ static('img/bcfp-logo_160x34@2x.png') }} 320w,
-                                 {{ static('img/bcfp-logo_160x34@3x.png') }} 480w,
-                                 {{ static('img/bcfp-logo_160x34@4x.png') }} 640w,
-                                 {{ static('img/bcfp-logo_240x50.png') }} 240w,
-                                 {{ static('img/bcfp-logo_240x50@2x.png') }} 480w"
-                         sizes="(max-width: 900px) 160px,
-                                240px"
-                         alt="Bureau of Consumer Financial Protection">
+                         src="{{ static('img/logo_237x50.png') }}"
+                         srcset="{{ static('img/logo_161x34.png') }} 161w,
+                                 {{ static('img/logo_161x34@2x.png') }} 322w,
+                                 {{ static('img/logo_161x34@3x.png') }} 483w,
+                                 {{ static('img/logo_161x34@4x.png') }} 644w,
+                                 {{ static('img/logo_237x50.png') }} 237w,
+                                 {{ static('img/logo_237x50@2x.png') }} 474w"
+                         sizes="(max-width: 900px) 161px,
+                                237px"
```

Comment:
Should this be on the line before?

Reply:
No. 237px is the default size, and 161px is the secondary size for smaller screens. Each size should be on its own line.